solr/solr-ref-guide/modules/indexing-guide/pages/luke-request-handler.adoc

```diff
@@ -17,7 +17,7 @@
 // under the License.
 
 The Luke Request Handler offers programmatic access to the information provided on the xref:schema-browser-screen.adoc[] page of the Admin UI.
-It is modeled after Luke, the Lucene Index Browser by Andrzej Bialecki.
+It is modeled after https://github.com/apache/lucene/tree/main/lucene/luke[Luke], the Lucene Index Browser.
```

Comment:
Luke has moved to a Lucene module since this was written, so I removed @sigram reference here, and instead provided a link to the module.

PS: Do we have a recipe for how users can run Luke standalone? Should we mention it here?

Reply:
possible nice-to-have: linking not to `main` but to specific Lucene release

Reply:
Do we have a ref-guide variable for current lucene version? If so, we could surely do that. Even if the user will not be able to download Luke from that location (or anywhere else for that matter), she needs to download Lucene's binary distro and run it from there.

Reply:
We can use `{dep-version-lucene}`

Reply:
Fixed